style/modules/testimonial.css

```diff
@@ -0,0 +1,83 @@
+/* Desktop */
+/* Background Box */
+
+.testimonial {
+    margin-top: 100px;
+    max-width: 610px;
+    background: white;
+    border: 0.5px solid black;
+}
+
+/* Testimonial Picture */
+
+.testimonial--pic {
+    position: relative;
+    width: 200px;
+    height: 200px;
+    left: 50%;
+    margin-left: -100px;
+    transform: translateY(-80px);
+    overflow: hidden;
+}
+
+.testimonial--pic > img {
```

Comment:
I believe you can remove everything in this block except for width, height, and border-radius without changing the behavior.

(having `object-fit: cover` will probably change the behavior if the image isn't already cropped square, but I'm guessing we'll want to do that manually anyway, because it's not a given for non-square images that what's dead center is what you want to preserve)